### DISCUSSION TITLE
SYCL or OneAPI Support for MFEM?

### DISCUSSION THREAD

**Discussion by jyoung3131** [mfem/mfem#2544] [Q&A]
Hi everyone,

I'm looking into the feasibility of running LAGHOS with Intel GPU and FPGA platforms as part of an ongoing project. Do you have any insight into whether there are any SYCL or OneAPI efforts to support MFEM for applications?

Thanks,
Jeff

**Comment by tzanio** [comment]
Hi Jeff,

@camierjs can elaborate, but there is an initial SYCL backend implementation in the [sycl](https://github.com/mfem/mfem/tree/sycl) branch.

Tzanio

**Reply by jyoung3131** [reply]
Great! We are testing with Intel's OneAPI (2021.3), so I will try to test compiling with this branch and toolchain combination.
